services/web/server/src/simcore_service_webserver/studies_dispatcher/_studies_access.py

```diff
@@ -211,6 +249,62 @@ async def copy_study_to_account(
 
 
 # HANDLERS --------------------------------------------------------
+
+
+class RedirectToFrontEndPageError(Exception):
+    def __init__(
+        self, human_readable_message: str, error_code: str, status_code: int
+    ) -> None:
+        self.human_readable_message = human_readable_message
+        self.error_code = error_code
+        self.status_code = status_code
+        super().__init__(human_readable_message)
+
+
+def _handle_errors_with_error_page(handler: Handler):
+    @functools.wraps(handler)
+    async def wrapper(request: web.Request) -> web.StreamResponse:
+        try:
+            return await handler(request)
+
+        except ProjectNotFoundError as err:
+            raise create_redirect_response(
+                request.app,
+                page="error",
+                message=compose_support_error_msg(
+                    msg=MSG_PROJECT_NOT_FOUND.format(project_id=err.project_uuid),
+                    error_code="PROJECT_NOT_FOUND",
+                ),
+                status_code=web.HTTPNotFound.status_code,
+            ) from err
+
+        except RedirectToFrontEndPageError as err:
+            raise create_redirect_response(
+                request.app,
+                page="error",
+                message=compose_support_error_msg(
+                    msg=err.human_readable_message, error_code=err.error_code
+                ),
+                status_code=err.status_code,
+            ) from err
+
+        except Exception as err:
+            error_code = create_error_code(err)
+            log.exception(
+                "Unexpected failure. TIP: too many simultaneous request? [%s]",
```

Comment:
I don't get the `TIP` message here. Would that help solve the exception that was raised?